migrating/cluster-wide/ossm-migrating-cluster-wide.adoc

```diff
@@ -92,14 +93,14 @@ You can proceed to complete the migration and remove {SMProduct} 2 resources.
 Before creating a default revision tag and relabelling the namespaces, you must migrate all remaining workload namespaces, including gateways.
 ====
 
-// The following tasks are for migration using the Istio injection label with cert-manager
+// This sets the context to build the tasks for cluster-wide migration using the Istio injection label with cert-manager
+:context: cw-injection-cm
```

Comment:
After the content that needs this context label, I would suggest resetting the context to `ossm-migrating-cluster-wide`. Someone looking to make a link to a heading further down the page will probably expect e.g. `ossm-migrating-workloads-using-the-simple-migration-method.adoc` to be using the assembly context, so it's helpful to keep the special context values scoped to only where you need them to change how the content renders

Reply:
done